### DISCUSSION TITLE
Password reset needed-ngnix proxy manager

### DISCUSSION THREAD

**Discussion by CustomerJFK** [tteck/Proxmox#2061] [Help Needed]
Hi 
I installed the helper script for the ngnix proxy manager and don`t save my password. 
How can I reset the password of the ngnix proxy manger from the CLI of my Proxmox LXC? (Which commands???)
It would be great if anyone can help soon.
Thanks

**Comment by Smile2Grow** [comment]
Enter

`bash -c "$(wget -qLO - https://github.com/tteck/Proxmox/raw/main/ct/paperless-ngx.sh)"
`

 in the console of the Paperless LXC container and query the Paperless-ngx Credentials  there.
